intellij-plugin-structure/structure-intellij/src/main/java/com/jetbrains/plugin/structure/intellij/problems/MinorWarnings.kt

```diff
@@ -155,21 +155,29 @@ class ElementMissingAttribute(
 }
 
 
-class SuspiciousUntilBuild(
-  private val untilBuild: String
+open class SuspiciousUntilBuild(
+  private val untilBuild: String,
+  private val additionalMessage: String = ""
 ) : PluginProblem() {
   override val hint = ProblemSolutionHint(
     documentationUrl = "https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html"
   )
   override val message: String
-    get() = "The <until-build> '$untilBuild' does not represent the actual build number. If you want your plugin to " +
-            "be compatible with all future IDEs, you can leave this field empty. However, we highly recommend " +
-            "setting it to the latest available IDE version."
+    get() = "The <until-build> '$untilBuild' does not represent the actual build number. " +
+            if (additionalMessage.isNotBlank()) "$additionalMessage " else "" +
+            "If you want your plugin to be compatible with all future IDE versions, you can remove this attribute. " +
+            "However, we highly recommend setting it to the latest available IDE version."
 
   override val level
     get() = Level.WARNING
```

Comment:
Shouldn't this be also `error` for new and `unacceptable warning` for existing plugins? Or is it for another issue and review?

Reply:
`SuspiciousUntilBuild` is now for builds like `281` (2028.1). I'd leave this as an warning as it is in the grey zone.